### ISSUE TITLE
Parsing error for workbench3.1 install

### ISSUE THREAD

**Issue by gareth-smyth** [sodero/InstallerLG#45]
The workbench3.1 install script seems to have a mistake in the following section that is handled differently in InstallerLG (I've removed some lines).

```
(set startupII        (tackon target "S/StartupII"))
(set cli-startup      (tackon target "S/CLI-Startup"))

(set oldfiles "")
(if (exists startupII)
(
    (set oldfiles (cat oldfiles startupII "�����������\n"))   ; these are hard spaces
))

(if (exists cli-startup)
(
    (set oldfiles (cat oldfiles s/cli-startup "���������\n"))   ; these are hard spaces
))
```

I think the second last line should be using the variable `cli-startup` instead of the unquoted string `s/cli-startup`.  

**Original installer**
The Install3.1 installer parses this okay, but will concatenate `s/cli-startup` as if it were an empty string

**InstallerLG**
InstallerLG fails with `Line 3616: syntax error '/'`

**Test script**
This script will output "AC" on the original install and a parse error in InstallerLG.

```
(set cli-startup "B")
(debug (cat "A" s/cli-startup "C"))
```

**Comment by sodero** [comment]
In this case I think InstallerLG is doing the right thing. It's obvious that this is a bug in the installer script that would have been found if the original Installer had a proper parser.

I've found similar problems with the MorphOS installer utility as well. It tolerates missing closing brackets in some places, but not in others (WHDLOpener and ghostscriptinstaller).

**Comment by gareth-smyth** [comment]
Okay that makes sense to me.

**Comment by sodero** [comment]
Do you know if this problem exists in 3.1.4 BTW? Since development seems to continue it might be worth it to notify the developers.

**Comment by sodero** [comment]
I notified Thomas Richter on EAB. Hopefully we'll get some feedback.

**Comment by sodero** [comment]
Minuous has acknowledged and fixed this in 3.2.
